modules/investigating-master-node-installation-issues.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+//
+// *support/troubleshooting-installations.adoc
+
+[id="investigating-master-node-installation-issues_{context}"]
+= Investigating master node installation issues
+
+If you experience master node installation issues, determine master node, software defined network (SDN) and network operator status. Collect `kubelet.service`, `crio.service` journald unit logs and master node container logs for visibility into master node agent, CRI-O container runtime and pod activity.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role. Note that the initial kubeadmin password can be found in `<install_directory>/auth/kubeadmin-password` on the installation host.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program the same SSH key.
+* If you tried to install a cluster on infrastructure that you provisioned, you must have the fully-qualified domain names of the control plane, or master, machines.
+
+.Procedure
+
+. Determine master node status:
```

Comment:
If your having installation issue you probably can't run many of these `oc` commands.

Reply:
Good point. I have now added a note in the investigating master node installation and invesgigating worker node installation modules highlighting this. I have also changed the ordering of the procedure a little so that we start with console monitoring and other elements which don't require the API and the kubelet to be running.